add_jackson_to_dist.sh

```diff
@@ -1,95 +0,0 @@
-#!/bin/bash -xe
```

Comment:
Let's do one thing at a time and revert these kind of changes. The goal of this issue is to fix the Jackson vulnerabilities. We can refactor where we add the Jackson library in a separate step.